### PR TITLE
feat: various changes for migration

### PR DESCRIPTION
**Pull request by colbr**
Pushing a whole bunch of changes to prep for initial PR into zOS.

### CODE REVIEW

File: src/features/view-subdomains/SubdomainMetrics/SubdomainMetrics.tsx [hunk 1]
```diff
@@ -6,49 +6,94 @@ import { DomainMetrics, TokenPriceInfo } from '@zero-tech/zns-sdk';
...
 export const SubdomainMetrics: FC<SubdomainMetricsProps> = ({
-	metrics,
+	domainId,
 	paymentTokenInfo,
 }) => {
+	const { data: metrics, isLoading } = useDomainMetrics(domainId);
+
+	const DEFAULT = { isLoading };
+
+	const paymentTokenLabel = paymentTokenInfo?.name
```
Comment (domw30) [inline src/features/view-subdomains/SubdomainMetrics/SubdomainMetrics.tsx:26]: nice these look much cleaner to expand on!

File: src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx [hunk 2]
```diff
@@ -31,74 +31,83 @@ export const SubdomainTable: FC<SubdomainTableProps> = ({
...
 		}
-	};
+	}, []);
 
-	const changeView = useCallback(
-		(isGridView: boolean) => {
-			setIsGridView(isGridView);
-		},
-		[setIsGridView],
+	const Row = useCallback(
```
Comment (domw30) [inline src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx:41]: Good shout moving these up 🧹

File: src/lib/constants/networks.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export enum NETWORK_TYPES {
 }
 
 const ENV_NETWORK = process.env.REACT_APP_DEFAULT_NETWORK;
-export const DEFAULT_NETWORK = Network.RINKEBY;
+export const DEFAULT_NETWORK = Network.MAINNET;
```
Comment (domw30) [inline src/lib/constants/networks.ts:13]: ☠️ Rinkeby

**Review by domw30 — APPROVED** [review]
Very nice! LGTM

File: src/lib/constants/networks.ts (hunk 3 above)
Comment (colbr) [reply src/lib/constants/networks.ts:13]: Sadge 😿

**Pull request merged**